07.bowling_object/frame.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require_relative './shot'
+
+class Frame
+  def initialize(first_mark, second_mark = nil, third_mark = nil)
+    @first_mark = Shot.new(first_mark)
+    @second_mark = Shot.new(second_mark)
+    @third_mark = Shot.new(third_mark)
```

Comment:
`Frame`クラスが`mark`まで扱う必要はないと思います。`mark`を扱うのは`Shot`クラスに任せましょう。
ここではシンプルに`Shot`オブジェクトをコンストラクタ引数として受け取る形にした方が良いと思います。

```suggestion
  def initialize(first_shot, second_shot = nil, third_shot = nil)
    @first_shot = first_shot
    @second_shot = second_shot
    @third_shot = third_shot
  end
```